templates/terraform/environments/stage/main.tf

```diff
@@ -41,6 +41,17 @@ data "terraform_remote_state" "shared" {
   }
 }
 
+# rds shared db password for dev envrionment
+module "rds_dev_secret" {
```

Comment:
Honestly it would probably be fine to just set this password to be the project name or something, but if it's in a secret could you make sure that the dev group can read this secret?

Reply:
yes, dev group can read it